templates/Pages/news.php

```diff
@@ -5,16 +5,29 @@
 
     <ul class="news">
         <li>
-            <span class="date">2019-11-19</span> The new IO500 list is published at SC'19.
+            <span class="date">2021-03-19</span> The <a class="link" href="https://io500.org/site/cfs">call for submissions</a> for the eighth IO500 list at ISC 2021 sent out.
         </li>
         <li>
-            <span class="date">2019-11-10</span> The schedule for our Supercomputing Birds-of-a-feather session is now defined
+            <span class="date">2020-11-18</span>The seventh IO500 list is published at SC'20.
         </li>
+        <li>
+            <span class="date">2020-07-23</span>The sixth IO500 list is published at ISC'20.
```

Comment:
Jean Luca, I was wondering how to add a link to the specific list pages for each of these news entries?  I copied the "call for submissions" link that was previously in use for the older news items, but IMHO that seems "wrong" to use a full URL for a page on the same site, especially since this URL will become incorrect with your upcoming changes.

Reply:
@adilger That makes sense! The correct way to add a link for the static pages would be like this:
```
echo $this->Html->link('call for submissions', [     // title for the link
    'controller' => 'Pages',
    'action' => 'view',
    'cfs'                                            // name of the file
]
```
The framework will then resolve for the full page URL and use any custom routes we may have defined as well.